cmd/cluster-agent/commands/check/check.go

```diff
@@ -11,17 +11,16 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```

Comment:
LGTM for `cmd/cluster-agent/commands/check/check.go`.
just a small nit regarding the imports order, I think external packages should come second after the standard library imports.

Reply:
Fixed. I had the wrong import sort set in the IDE.